tensorflow/core/kernels/cwise_op_zeta.cc

```diff
@@ -19,7 +19,7 @@ namespace tensorflow {
 REGISTER2(BinaryOp, CPU, "Zeta", functor::zeta, float, double);
 REGISTER2(BinaryOp, CPU, "Polygamma", functor::polygamma, float, double);
 
-#if GOOGLE_CUDA
+#if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
```

Comment:
@jeffdaily from git log GPU support for zeta was added on 8/31/2018 and from the implementation of `functor::zeta` it depends on `Eigen`. Since corresponding test is failing on ROCm path I think it's prudent to NOT add `TENSORFLOW_USE_ROCM` here.

Reply:
Do you think it's worth trying to enable the polygamma on ROCm path, while leaving the zeta CPU-only?